### PR TITLE
Update documentation for offline installation. 

### LINKED ISSUE #1741: Document offline installation of QPC
## Feature description:
Create a section for offline QPC installation in https://quipucords.readthedocs.io/en/latest/ for QPC.

This should cover flags required by install.sh and locations for rpms/docker images.

### PR DESCRIPTION
**Pull request by myersCody**
Closes #1741

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Installing and Configuring Quipucords
 =====================================
-Quipucords is delivered in two parts, a command line tool as an RPM package and a server as a container image. The following instructions describe how to install the parts of Quipucords by using a scripted installation or by using step-by-step commands.
+Quipucords is delivered in three parts, a command line tool as an RPM package and the server and database as a container image. The following instructions describe how to install the parts of Quipucords by using a scripted installation or by using step-by-step commands.
```
Comment (abaiken) [inline docs/source/install.rst]: Change the first sentence to: `Quipucords is delivered in two parts, a command line tool as an RPM package and a server as container images.`

File: docs/source/install_scripted.rst [hunk 2]
```diff
@@ -46,26 +36,37 @@ The output appears similar to the following example::
 
     Installation complete.
 
-Next Steps
-++++++++++
-When the installation completes, the Quipucords server is installed and started. In addition, a connection to the command line tool is configured on the same system on which the server is installed. However, you must still complete the following steps before you can begin using Quipucords:
+Installing without Internet Connectivity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you choose the offline option to run the installer, these associated dependencies must be installed.
```
Comment (abaiken) [inline docs/source/install_scripted.rst]: change these to `the following`

File: docs/source/install_scripted.rst [hunk 3]
```diff
@@ -46,26 +36,37 @@ The output appears similar to the following example::
...
-To install the command line tool, you must have the following items. For more information about download locations for specific operating systems, see `Installing the Quipucords Command Line Tool <install.html#commandline>`_.
+- Server Container Image (quipucords.version.tar.gz)
+- Postgres Container Image (postgres.version.tar.gz)
+- **Note:** You must extract the tarball to access the postgres image with the command::
+
+  # tar -xzvf postgres.version.tar.gz
+
+**Command Line Tool RPM Package:**
+
+- QPC Package (all of the qpc-version.rpm)
```
Comment (abaiken) [inline docs/source/install_scripted.rst]: break these out into the different os versions

File: docs/source/install_scripted.rst [hunk 4]
```diff
@@ -1,19 +1,9 @@
...
-
-When you run the scripted installation, the server is installed and started as described in `Configuring and Starting Quipucords <install.html#config-and-start>`_. However, you can also run the scripted installation with options if you want to change some of the defaults.
-
-Installing the Ansible Prerequisite
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Prepare the system to install the software by installing Ansible. For more information, see the `Ansible installation documentation <http://docs.ansible.com/ansible/latest/intro_installation.html#installing-the-control-machine>`_.
+The scripted installation runs an installer that uses Ansible to install the command line tool, quipucords server image, and the database image. When you run the scripted installation, the server is installed and started as described in `Configuring and Starting Quipucords <install.html#config-and-start>`_. However, you can change some of the defaults used by the scripted installation with the `Installation Options <install.html#installation-options>`_ .
```
Comment (abaiken) [inline docs/source/install_scripted.rst]: For some reason the `Installation Options` link is not working here.

File: docs/source/install_scripted.rst [hunk 5]
```diff
@@ -1,19 +1,9 @@
...
-
-When you run the scripted installation, the server is installed and started as described in `Configuring and Starting Quipucords <install.html#config-and-start>`_. However, you can also run the scripted installation with options if you want to change some of the defaults.
-
-Installing the Ansible Prerequisite
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Prepare the system to install the software by installing Ansible. For more information, see the `Ansible installation documentation <http://docs.ansible.com/ansible/latest/intro_installation.html#installing-the-control-machine>`_.
+The scripted installation runs an installer that uses Ansible to install the command line tool, quipucords server image, and the database image. When you run the scripted installation, the server is installed and started as described in `Configuring and Starting Quipucords <install.html#config-and-start>`_. However, you can change some of the defaults used by the scripted installation with the `Installation Options <install.html#installation-options>`_.
```
Comment (myersCody) [inline docs/source/install_scripted.rst]: Installation Options hyperlink is now currently working.

Comment (myersCody) [reply docs/source/install_scripted.rst]: It is now working.

File: docs/source/install_steps.rst [hunk 6]
```diff
@@ -14,3 +18,10 @@ The Quipucords server is delivered as a container image that runs on your server
 .. include:: install_server_fedora.rst
 .. include:: install_container_image.rst
 .. include:: configure.rst
+
+Configuring Quipucords
```
Comment (abaiken) [inline docs/source/install_steps.rst]: I think that we should remove this section entirely. If you view it in readthedocs you can see that the section ```Changing the Default Password for ...``` is literally above it, as is a link to ```Logging in to the Quipucords Server```.

Comment (kholdaway) [reply docs/source/install_steps.rst]: I agree with Ashley

**Comment by kholdaway** [comment]
Looks great @myersCody!

**Pull request merged**